js_modules/dagit/packages/core/src/assets/LaunchAssetChoosePartitionsDialog.tsx

```diff
@@ -372,6 +397,11 @@ const LaunchAssetChoosePartitionsDialogBody: React.FC<Props> = ({
                   ),
                 )
               }
+              partitionDefinitionName={
```

Comment:
Also, not sure about this since after we spoke you said there could be multiple partition definition names?

Reply:
I think this can be `displayedPartitionDefinition?.name` actually 🙌